kitsune/users/migrations/0006_auto_update_help_text_email_sent_fields.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import kitsune.sumo.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0005_set_initial_contrib_email_flag'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='profile',
+            name='first_l10n_email_sent',
+            field=models.BooleanField(default=False, help_text='Has been sent a first revision contribution email.'),
```

Comment:
You changed the help text on two boolean fields, but this is still a string and not unicode and there's only one boolean field represented. That seems fishy.